### ISSUE TITLE
Dashlinks: Bug with dashboard with same name on different folders

### ISSUE THREAD

**Issue by tecmatia-dp** [grafana/grafana#12523]
Hi team!
An issue with folders and dashboards with same name (very usefull! thanks!)

### What Grafana version are you using?
5.2.1
### What did you do?
Have panels with same name in different folders. Use dashboard links...
### What was the expected result?
Any link should redirect to correct folder, despite dashboards with same name...
### What happened instead?
Links to panels with same name, redirects to other with same name in other folder...

I think href in dashboard links refers to old URL format, without folder info, so it opens first with that name...

**Comment by tecmatia-dp** [comment]
I fixed it, changing this line in "\public\app\features\dashlinks\module.ts", from:

```
              memo.push({
                title: dash.title,
                url: 'dashboard/' + dash.uri,
                target: link.target,
                icon: 'fa fa-th-large',
                keepTime: link.keepTime,
                includeVars: link.includeVars,
              });
```
to:

```
              memo.push({
                title: dash.title,
                url: dash.url,
                target: link.target,
                icon: 'fa fa-th-large',
                keepTime: link.keepTime,
                includeVars: link.includeVars,
              });
```

I'm not familiar with git... so I do not risk a pull request...apologies!

**Comment by torkelo** [comment]
related to this issue, https://github.com/grafana/grafana/issues/10705

**Comment by marefr** [comment]
Fixed by #12844
